### PR TITLE
Fix bug where having no excludes for uninstall would trigger a panic

### PR DESCRIPTION
**Pull request by jamesc**
Signed-off-by: James Casey <james@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/hab/src/main.rs [hunk 1]
```diff
@@ -1363,11 +1363,13 @@ fn install_sources_from_matches(matches: &ArgMatches) -> Result<Vec<InstallSourc
 }
 
 fn excludes_from_matches(matches: &ArgMatches) -> Vec<PackageIdent> {
-    matches
-        .values_of("EXCLUDE")
-        .unwrap() // Required via clap
```
Comment (baumanj) [inline components/hab/src/main.rs]: I think a simpler fix would be changing the `unwrap` to `unwrap_or_default`.

Comment (jamesc) [reply components/hab/src/main.rs]: You're a master of the clap API :)  Done.

Comment (baumanj) [reply components/hab/src/main.rs]: FYI, that's not clap-specific, rather a feature of [`Option`](http://doc.rust-lang.org/1.30.0/core/option/enum.Option.html#method.unwrap_or_default).

**Pull request merged**